tests/functional/oauth_query_param_validation.js

```diff
@@ -60,11 +60,8 @@ var testErrorInclude = function (expected) {
 
 /*eslint-disable camelcase */
 registerSuite('oauth query parameter validation', {
-  beforeEach: function () {
+  before: function () {
```

Comment:
`before` in these tests is really annoying with `--grep`. A `before` will be run whether any test in this suite matches `--grep=XXX` or not. Can we do this in the beforeEach somehow, check whether the CLIENT_ID's are set and if not, do this?